index.html

```diff
@@ -145,6 +145,11 @@ <h2>
         For example, the badge content '7' should be displayed as '7' in the
         <a data-cite="ltli/#locale">locale</a> 'en-NZ' but as '٧' in 'ar-EG'.
       </p>
+      <p>
+        Note: The user agent is MAY clear the badge at any time, though this
+        should generally only be done in extreme circumstances, such as
+        clearing site data, or restarting the system.
```

Comment:
I know this is just a non-normative note, but "restarting the system" doesn't sound like a great suggestion tbh. Ideally, the badge state is retained across OS restarts. Badges definitely keep their numbers across restarts on iOs and, I believe, MacOS. 
```suggestion
        clearing site data.
```